bootstrap/bootstrap.php

```diff
@@ -10,12 +10,7 @@
 $request = $app->router->match($GLOBALS, $_SERVER);
 
 try {
-    $page = $app->resource
-        ->{$request->method}
-        ->uri($request->path)
-        ->withQuery($request->query)
-        ->eager
-        ->request();
+    $page = $app->resource->{$request->method}->uri($request->path)($request->query);
```

Comment:
Probably the current one looks better less code. But not sure how this is working. :) .

`uri($request->path)` is this returning an invokable ?